spec/support/helpers/feature_helpers.rb

```diff
@@ -61,7 +61,7 @@ def answer_gyr_triage_questions(screenshot_method: nil, **options)
           TaxReturn.current_tax_year - 2,
           TaxReturn.current_tax_year - 1,
         ],
-        assistance_options: ['chat'],
```

Comment:
It's simpler to select only one option here, e.g. `phone_review_english` (which is probably the common case). Not a blocker.